fvm/evm/stdlib/contract.cdc

```diff
@@ -224,6 +238,7 @@ contract EVM {
                 from: self.addressBytes,
                 amount: balance.attoflow
             ) as! @FlowToken.Vault
+            emit FLOWTokenWithdraw(addressBytes: self.addressBytes, amount: balance.inFLOW())
```

Comment:
I would not emit an event if the balance is 0. I think I would prefer the whole function just exiting early if the balance is 0.

same with deposit.

Reply:
added protection